### DISCUSSION TITLE
how do I get T-shirt in Hacktober

### DISCUSSION THREAD

**Discussion by Shravanllll** [theonlyanson/Hacktoberfest-2022#160] [Q&A] [answered]
how do I get T-shirt in Hacktober

**Comment by theonlyanson** [comment, marked as answer]
> how do I get T-shirt in Hacktober

you need to make 4 pull requests which a maintainer should merge and label as hacktoberfest-accepted then there will be 7 days given to the maintainers to review the contributions and label as invalid or spam. If the maintainer labels your contributions as spam then you will have to create one more valid pull request. If the maintainer marks 2 or more contribution as invalid or spam then your account will be disqualified from hacktoberfest.
